tests/generate.py

```diff
@@ -32,10 +33,10 @@
 NOT_AFTER: datetime.datetime = datetime.datetime.utcfromtimestamp(0x1FEDF00D + 30)
 
 ANY_PRIV_KEY = Union[
-    ed25519.Ed25519PrivateKey | ec.EllipticCurvePrivateKey | rsa.RSAPrivateKey
+    ed25519.Ed25519PrivateKey, ec.EllipticCurvePrivateKey, rsa.RSAPrivateKey
 ]
 ANY_PUB_KEY = Union[
-    ed25519.Ed25519PublicKey | ec.EllipticCurvePublicKey | rsa.RSAPublicKey
+    ed25519.Ed25519PublicKey, ec.EllipticCurvePublicKey, rsa.RSAPublicKey
 ]
```

Comment:
Ah interesting! I see in [the docs](https://docs.python.org/3/library/typing.html#typing.Union) that: "Changed in version 3.10: Unions can now be written as X | Y. See union type expressions."

Ddid you change this for concerns about older Python versions? Or is one form is more common in practice than another? I'm ok with the change in either case, just want to understand the motivation.

Reply:
IMO this change should be separately motivated/in a separate commit? It seems unrelated to "Adding integration tests".

Reply:
Main reason is that Pylance gave me the following error 
```
Union requires two or more type argumentsPylance[reportInvalidTypeArguments](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportInvalidTypeArguments)
```

But since it has not that much to do with the goal of this PR it might be better to keep it out?! If you want this change I'll add it to a different commit.